library/src/blas1/rocblas_copy.cpp

```diff
@@ -62,24 +52,10 @@ namespace
 
         RETURN_ZERO_DEVICE_MEMORY_SIZE_IF_QUERIED(handle);
 
-        // Quick return if possible.
-        if(n <= 0)
-            return rocblas_status_success;
-
-        int  blocks = (n - 1) / NB + 1;
-        dim3 grid(blocks);
-        dim3 threads(NB);
-
-        hipStream_t rocblas_stream = handle->rocblas_stream;
+        if(!incx || !incy)
```

Comment:
`incx ==0 || incy==0` should be valid but not tested for -- see email thread and: http://www.netlib.org/lapack/lapack-3.1.1/html/dcopy.f.html

Reply:
must've slipped by, will correct that.